lib/kube/proxy/sess.go

```diff
@@ -653,6 +653,8 @@ func (s *session) launch(ephemeralContainerStatus *corev1.ContainerStatus) (retu
 		KubernetesPodMetadata:     eventPodMeta,
 		InitialCommand:            q["command"],
 		SessionRecording:          s.ctx.recordingConfig.GetMode(),
+		Invited:                   strings.Split(q.Get(teleport.KubeSessionInvitedQueryParam), ","),
+		Reason:                    q.Get(teleport.KubeSessionReasonQueryParam),
```

Comment:
This info is already pulled into the session object so lets reuse it 
```suggestion
		Invited:                   s.invited,
		Reason:                  s.reason,
```